### PR TITLE
Add japanese translation

### PR DESCRIPTION
**Pull request by kuragehimekurara1**

### CODE REVIEW

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <!-- About -->
+    <string name="source_code">ソースコード</string>
+    <string name="lyrics_sources">歌詞の出典</string>
+
+    <string name="source_genius" translatable="false">Genius</string>
+    <string name="source_deezer" translatable="false">Deezer</string>
+
+    <string name="source_code_url" translatable="false">https://github.com/TecCheck/FastLyrics</string>
+    <string name="source_url_genius" translatable="false">https://genius.com/</string>
+    <string name="source_url_deezer" translatable="false">https://deezer.com/</string>
```
Comment (TecCheck) [inline app/src/main/res/values-ja/strings.xml:61]: You don't need to have the strings with `translatable="false"` in your translation file

**Review by TecCheck — COMMENTED** [review]
Thanks a lot for the translation. There's only one minor detail to change. I've added a comment on it. Would be cool if you could fix that

File: app/src/main/res/values-ja/strings.xml (hunk 1 above)
Comment (kuragehimekurara1) [reply app/src/main/res/values-ja/strings.xml]: Thank you for waiting.
The correction has been completed.
Is this OK?

**Comment by TecCheck** [comment]
This looks great. Thank you very much :+1:

**Pull request merged**